packages/devtools/src/index.js

```diff
@@ -46,17 +46,10 @@ export default class DevTools {
         }
 
         sessionMap.set(sessionId, { browser, session: driver })
-        const environmentPrototype = {
-            isDevTools: { value: true },
-            isW3C: { value: true },
-            isMobile: { value: false },
-            isIOS: { value: false },
-            isAndroid: { value: false },
-            isChrome: { value: browserName === 'chrome' },
-            isSauce: { value: false },
-            isSeleniumStandalone: { value: false },
-            getPuppeteer: { value: /* istanbul ignore next */ () => browser }
-        }
+        const environmentPrototype = { getPuppeteer: { value: /* istanbul ignore next */ () => browser } }
```

Comment:
I think it won't work like this, because we need to have an object where flag is wrapped with { `value: ...` }:
```
{
  getPuppeteer: { value: () => browser },
  isChrome: { value: true },
  ...
```
but not
```
{
  getPuppeteer: { value: () => browser },
  isChrome: true,
  ...
```